framework/src/engine/legacy/legacy_chain_handler.ts

```diff
@@ -15,6 +15,7 @@
 import { codec } from '@liskhq/lisk-codec';
 import { P2PRequestPacket } from '@liskhq/lisk-p2p/dist-node/types';
 import { Database, NotFoundError } from '@liskhq/lisk-db';
+import { INVALID_PEER_INFO_PENALTY } from '@liskhq/lisk-p2p/dist-node/constants';
```

Comment:
wrong import and maybe we should simply use 100 penalty or declare a constant within legacy to use